git-hooks/pre-commit.sh

```diff
@@ -0,0 +1,40 @@
+#!/bin/bash
```

Comment:
This is not a fool-proof solution, so should not be depended on too heavily. For example, I use the GitX GUI client to do some of my git work (I find it much better than the command line for reviewing changes and making commits). Unfortunately it does not respect/run git hooks.